test-benchmark-visualizer/test-benchmark-sparkline.html

```diff
@@ -0,0 +1,253 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<script src="../../d3/d3.js"></script>
+<link rel="import" href="../../paper-styles/paper-styles.html">
+<link rel="import" href="../../iron-resizable-behavior/iron-resizable-behavior.html">
+
+<!--
+This element visualizes the data available in a provide `PerformanceTest`
+instance. The current average and standard deviation of all available test
+run results is displayed at the top of the sparkline, and a series of columns
+are displayed to demonstrate the relative difference of each test run result.
+
+### Styling
+
+Custom property                           | Description                                      | Default
+------------------------------------------|--------------------------------------------------|------------------------
+`--test-benchmark-sparkline-light`        | Background color at the top of the sparkline.    | `--paper-blue-grey-500`
+`--test-benchmark-sparkline-dark`         | Background color at the bottom of the sparkline. | `--paper-blue-grey-700`
+`--test-benchmark-sparkline-axis-color`   | The axis color for the sparkline.                | `--paper-blue-grey-500`
+`--test-benchmark-sparkline-column-color` | The column color for the sparkline.              | `--paper-blue-grey-100`
+
+@demo demo/index.html
+-->
+
+<dom-module id="test-benchmark-sparkline">
+  <template>
+    <style>
+      :host {
+        @apply(--layout);
+        @apply(--layout-flex);
```

Comment:
Same comment as before, I don't think you want `--layout-flex`, and I don't think you need `--layout`